nominations/models.py

```diff
@@ -617,6 +617,12 @@ def save(self, *args, **kwargs):
         super(Application, self).save(*args, **kwargs)
 
     class Meta:
+        permissions = (
```

Comment:
Nice.

Reply:
Yeah I think we should do more of this. We have lots of hardcoded access control and some of it should probably use permissions instead.